prt_partner_tweaks_access/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+{
+    "name": "Partner Access Tweaks",
+    "summary": "This allows to restrict contact access based on user settings",
```

Comment:
Restrict access to contacts based on user settings

Reply:
Thanks, I'll fix it